conseil-common/src/main/scala/tech/cryptonomic/conseil/common/ethereum/Views.scala

```diff
@@ -36,11 +36,11 @@ trait Views {
   lazy val AccountsViewSql: SqlAction[Int, NoStream, Effect] = sqlu"""
     CREATE OR REPLACE VIEW ethereum.accounts AS
       SELECT
-        "to" AS address,
-        SUM(value) AS value
+        destination AS address,
+        SUM(amount) AS value
```

Comment:
maybe `balance` would be better name instead of value - WDYT?

Reply:
I'll change this to `balance` when I merge accounts view, contracts and tokens tables into `accounts` table